src/RichText.js

```diff
@@ -34,6 +38,8 @@ const MARKDOWN_REGEX = {
 
 const USERNAME_REGEX = /@\S+:\S+/g;
 const ROOM_REGEX = /#\S+:\S+/g;
+let EMOJI_REGEX = null;
+window.EMOJI_REGEX = EMOJI_REGEX = new RegExp(emojione.unicodeRegexp, 'g');
```

Comment:
Why does this need to be added to `window`? And why the different one in `src/autocomplete/EmojiProvider.js`?